website/projects/spoke/src/assets/datasets/index.json

```diff
@@ -13,13 +13,13 @@
   },
   {
     "id": "cad",
-    "name": "coronary artery disease",
+    "name": "Coronary artery disease",
```

Comment:
These need to be lowercase. The map uses the exact labels to search for nodes and create markers. The logic could probably be updated to support this, but please be sure that it does.